x/slash/types/params.go

```diff
@@ -30,6 +31,7 @@ var (
 	KeyEnableSlash               = []byte("EnableSlash")
 	KeySignedBlocksWindow        = []byte("SignedBlocksWindow")
 	KeyPenaltyDelegatorSize      = []byte("PenaltyDelegatorSize")
+	KeyPenaltyLifeSpan           = []byte("PenaltyLifeSpan")
```

Comment:
Change to a more common name: `PenaltyTimeout` with a comment on in unit of EthBlkNum